test/test_core.py

```diff
@@ -9066,6 +9066,18 @@ def test_pthread_create_embind_stack_check(self):
     self.emcc_args += ['-lembind']
     self.do_run_in_out_file_test('core/pthread/create.cpp')
 
+  @no_asan('asyncify stack operations confuse asan')
+  @no_wasm64('TODO: asyncify for wasm64')
+  @node_pthreads
+  def test_pthread_asyncify(self):
+    # We had a infinite recursion bug when enabling PTHREADS_DEBUG + ASYNCIFY.
+    # This was because PTHREADS_DEBUG calls back into WebAssembly for each call to `err()`.
+    self.set_setting('PTHREADS_DEBUG')
+    self.set_setting('ASYNCIFY')
+    self.set_setting('PTHREAD_POOL_SIZE', 2)
+    self.set_setting('EXIT_RUNTIME')
+    self.do_run_in_out_file_test('core/pthread/asyncify.c')
```

Comment:
This feels maybe excessive for `core`, that is, to run it in each opt mode separately - since it's just for `PTHREADS_DEBUG`. How about `other`?

Reply:
Sure. Done.